examples/access-control-migration/package.json

```
@@ -28,8 +28,8 @@
     "test": "lb-mocha \"dist/__tests__/**/*.js\"",
     "test:dev": "lb-mocha --allow-console-logs dist/__tests__/**/*.js",
     "verify": "npm pack && tar xf loopback-access-control-migration*.tgz && tree package && npm run clean",
-    "migrate": "node ./dist/migrate",
-    "openapi-spec": "node ./dist/openapi-spec",
+    "migrate": "npm run build && node ./dist/migrate",
```

Have you considered using a `premigrate` script instead?

```suggestion
    "premigrate": "npm run build",
    "migrate": "node ./dist/migrate",
```

See how we are using `pretest` and `prestart` to run `npm run build` before the tests, and [Pre & Post Scripts](https://docs.npmjs.com/using-npm/scripts#pre--post-scripts) in npmjs docs.

(Same comment applies to `openapi-spec`, I think the script would be called `preopenapi-spec`.)

---

@bajtos thanks, done.

---

@bajtos looks good now?